Lib/keyword.py

```diff
@@ -56,7 +56,8 @@
 softkwlist = [
     '_',
     'case',
-    'match'
+    'match',
+    'type'
```

Comment:
So, now `type` is soft keyword. 
Should it be tested in `test_keyword`?
Presense of `match`, `_`, `case` in `softkwlist` currently tested.

Reply:
Added